pkg/ctl/install/install.go

```diff
@@ -138,12 +140,15 @@ func newFluxInstaller(rc *cmdutils.ResourceCmd, opts *installFluxOpts) (*fluxIns
 	if err != nil {
 		return nil, errors.Errorf("cannot create Kubernetes client set: %s", err)
 	}
+
+	gitClient := git.NewGitClient(ctx, opts.templateParams.GitUser, opts.templateParams.GitEmail, opts.timeout)
```

Comment:
Nit: we could pass a `GitClientParams` (or so) `struct` to group all git-related configuration & options.

Reply:
since they are passed only once to a constructor of the git.Client struct I will leave them for now